elements/pfelement/src/pfelement.js

```diff
@@ -32,7 +52,14 @@ class PFElement extends HTMLElement {
   }
 
   static get observedAttributes() {
-    return ["pfe-theme"];
+    const properties = this.properties;
+    if (properties) {
+      const oa = Object.keys(properties)
+        .filter(prop => properties[prop].observer || properties[prop].cascade || properties[prop].alias)
+        .map(p => this._prop2attr(p));
+      // console.log("observed attributes are", oa);
+      return [`${this.globalAttrPrefix}theme`, ...oa];
```

Comment:
Can we have a separate place to store an array of global attributes? Ideally we could define those in a properties function just like components do and have them join?

Reply:
Actually, I found a way for the parent and child classes to both use `static get properties()`. :+1: